build.ninja

```diff
@@ -1,7 +1,7 @@
 extensions=tex_math_dollars+mmd_title_block+link_attributes
-opts=-t html5 -f markdown_github+$extensions -H magic.head -s -c pandoc.css --mathjax -V lang=en
+opts=-t html5 -f markdown_github+$extensions -H magic.head -s -c static/pandoc.css --mathjax -V lang=en
 
 rule pd
   command = pandoc $in -o $out $opts
 
-build onion.html: pd onion.md | magic.head
+build onion.html: pd src/onion.md | magic.head
```

Comment:
And there will be a lot of .html inside the root directory

Reply:
So the url is `https://tox-rs.github.io/tox-human/onion.html`. Maybe htmls should be moved in a separate branch, but for now having them in the root is fine.